dunamai/__init__.py

```diff
@@ -978,6 +978,7 @@ def get_version(
     first_choice: Callable[[], Optional[Version]] = None,
     third_choice: Callable[[], Optional[Version]] = None,
     fallback: Version = Version("0.0.0"),
+    ignore: Union[str, Version] = "0",
```

Comment:
For backwards compatibility and more flexibility, I think `ignore` should be `Sequence[Union[str, Version]] = None`. We don't want to start ignoring something we didn't before.

Reply:
Done